docs/user-guide/tbl/tbl-data-reduction.ipynb

```diff
@@ -0,0 +1,236 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ngem['dim_0', 0].hist().plot(aspect='equal') + ngem['dim_0', 1].hist().plot(aspect='equal')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8",
+   "metadata": {},
+   "source": [
+    "### Time-of-flight"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wf.visualize(TofData[SampleRun], graph_attr={\"rankdir\": \"LR\"})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "10",
+   "metadata": {},
+   "source": [
+    "#### Cache the time-of-flight lookup table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "11",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "table = wf.compute(time_of_flight.TimeOfFlightLookupTable)\n",
+    "table.squeeze().plot()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wf[TimeOfFlightLookupTable] = table"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "13",
+   "metadata": {},
+   "source": [
+    "#### Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ngem_tofs = wf.compute(TofData[SampleRun])\n",
+    "ngem_wavs = wf.compute(WavelengthData[SampleRun])\n",
+    "\n",
+    "ngem_tofs.bins.concat().hist(tof=200).plot() + ngem_wavs.bins.concat().hist(wavelength=200).plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15",
+   "metadata": {},
+   "source": [
+    "## He3 detector data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "16",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "he3_wf = wf.copy()\n",
+    "\n",
+    "he3 = {}\n",
+    "for bank in ('he3_detector_bank0', 'he3_detector_bank1'):\n",
+    "    he3_wf[NeXusDetectorName] = bank\n",
+    "    he3[bank] = he3_wf.compute(DetectorData[SampleRun])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lims = {'vmin': 0, 'vmax': 4500}\n",
+    "\n",
+    "he3['he3_detector_bank0'].hist().plot(title='he3_bank0', **lims) + he3['he3_detector_bank1'].hist().plot(title='he3_bank1', **lims)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18",
+   "metadata": {},
+   "source": [
+    "### Time-of-flight"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "he3_tofs = {}\n",
+    "he3_wavs = {}\n",
+    "\n",
+    "for bank in ('he3_detector_bank0', 'he3_detector_bank1'):\n",
+    "    he3_wf[NeXusDetectorName] = bank\n",
+    "    he3_tofs[bank] = he3_wf.compute(TofData[SampleRun]).bins.concat().hist(tof=200)\n",
```

Comment:
(Minor) 
Should we decrease `tof` bin edges since it's just on a small dataset for documentation...?
It'll look prettier : D...